docs/vNext/Rate_Controllers.md

```diff
@@ -59,6 +60,26 @@ The PID rate controller, targeting a backlog of 5 transactions with a starting T
 }
 ```
 
+## Maximum Rate
+The maximum rate controller is a controller for driving the workers to their maximum achievable rate. This controller will aim to maximize the driven TPS for the worker by ramping up the driven TPS and backing off again when a drop in TPS is witnessed.
```

Comment:
"maximum achievable rate without overloading the SUT" perhaps? Just to clarify that this is a smart controller :)